plugins/social_network/include/golos/plugins/social_network/tag/tag_visitor.hpp

```diff
@@ -69,6 +69,10 @@ namespace golos { namespace plugins { namespace social_network { namespace tags
         template<typename Op>
         void operator()(Op&&) const {
         } /// ignore all other ops
+
+        // Helper method to extract metadata from comment_api_object
+        // removes empty tags and lowers case for the rest
+        static comment_metadata get_metadata(const comment_api_object &c);
```

Comment:
`get_metadata()`  is declared in `tags_object.hpp`. 
why do you move it here?

Reply:
Yes. My bad. I'll revert this.